### PR TITLE
Add Faq section

### PR DESCRIPTION
**Pull request by boooocchi**
### Overview
Added FAQ section below the info section using shadCN.

### Screenshots 
Screenshot 2024-12-02 at 14 41 02
Screenshot 2024-12-02 at 14 41 11

### CODE REVIEW

File: src/app/page.tsx [hunk 1]
```diff
@@ -173,7 +174,7 @@ export default function Home() {
 
       {/* Venue Information */}
       <section className="py-20 bg-gradient-to-b from-black to-gray-900">
-        <div className="container mx-auto px-4">
+        <div className="container mx-auto px-4 xl:max-w-[1100px]">
```
Comment (boooocchi) [inline src/app/page.tsx:177]: In order to align Info section and FAQ section width.

File: src/components/faq-accordion.tsx [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+  ];
+
+  return (
+    <Accordion type="single" collapsible className="w-full mx-auto">
+      {FAQs.map((qa, i) => (
+        <AccordionItem value={`item-${i + 1}`} key={i}>
+          <AccordionTrigger
+            className={`hover:text-green-500 transition-colors ${
+              openAccordionIndex === i && "text-green-500"
+            }`}
```
Comment (boooocchi) [inline src/components/faq-accordion.tsx:91]: when opened, the text color of the question will be green.

File: src/components/faq-accordion.tsx [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+          >
+            Instagram
+          </Link>{" "}
+          and videos here on{" "}
+          <Link
+            href="https://www.youtube.com/@PublicCloudNinja/about"
+            className="underline"
+          >
+            YouTube
+          </Link>
```
Comment (boooocchi) [inline src/components/faq-accordion.tsx:77]: Link to IG and YT

File: src/app/globals.css [hunk 4]
```diff
@@ -26,11 +24,11 @@
     --input: 210 40% 56%;
     --ring: 210 40% 60%;
     --radius: 1rem;
+    scroll-behavior: smooth;
```
Comment (boooocchi) [inline src/app/globals.css:27]: For a smooth scrolling.

Comment (bibschan) [reply src/app/globals.css:27]: nice touch!!

**Pull request merged**